File: PHP/PetShop.php
<?php

class PetShop {
    
    private $id;         
    private $name;       
    private $category;   
    private $price;      
    private $stock;      
    private $image;      
    
    public function __construct($id = -1, $name = "", $category = "", $price = 0, $stock = 0, $image = "") {
        $this->id = $id;
        $this->name = $name;
        $this->category = $category;
        $this->price = $price;
        $this->stock = $stock;
        $this->image = $image;
    }
    
    public function setData($id, $name, $category, $price, $stock, $image) {
        $this->id = $id;                 
        $this->name = $name;             
        $this->category = $category;     
        $this->price = $price;           
        $this->stock = $stock;           
        $this->image = $image;           
    }
    
    public function getId() {
        return $this->id;        
    }
    
    public function getName() {
        return $this->name;      
    }
    
    public function getCategory() {
        return $this->category;  
    }
    
    public function getPrice() {
        return $this->price;     
    }

    public function getStock() {
        return $this->stock;     
    }

    public function getImage() { 
        return $this->image;     
    }
}

?>

File: Python/PetShop.py
class PetShop:
    
    __id = -1          
    __name = ""        
    __category = ""    
    __price = 0        
    __stock = 0        

    def __init__(self, id = -1, name = "", category = "", price = 0, stock = 0):
        
        self.__id = id
        self.__name = name
        self.__category = category
        self.__price = price
        self.__stock = stock
    
    def set_data(self, id, name, category, price, stock):
        self.__id = id
        self.__name = name
        self.__category = category
        self.__price = price
        self.__stock = stock
    
    def get_id(self):
        return self.__id
    
    def get_name(self):
        return self.__name
    
    def get_category(self):
        return self.__category
    
    def get_price(self):
        return self.__price
    
    def get_stock(self):
        return self.__stock